event-bridge/src/test/kotlin/com/amplitude/eventbridge/EventBridgeTest.kt

```diff
@@ -0,0 +1,55 @@
+package com.amplitude.eventbridge
+
+import org.junit.jupiter.api.Assertions.assertEquals
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class EventBridgeTest {
+
+    @Test
+    fun `test addEventListener, sendEvent, listener called`() {
+        val testEvent = Event("test")
+        val identifyEvent = Event("identify")
+        val eventBridge = EventBridgeImpl()
+        eventBridge.setEventReceiver(
+            EventChannel.EVENT,
+            object : EventReceiver {
+                override fun receive(channel: EventChannel, event: Event) {
+                    assertEquals(event, testEvent)
+                }
+            }
+        )
+        eventBridge.setEventReceiver(
+            EventChannel.IDENTIFY,
+            object : EventReceiver {
+                override fun receive(channel: EventChannel, event: Event) {
+                    assertEquals(event, identifyEvent)
+                }
+            }
+        )
+        eventBridge.sendEvent(EventChannel.EVENT, testEvent)
+        eventBridge.sendEvent(EventChannel.IDENTIFY, identifyEvent)
```

Comment:
This could pass if the EventReceiver never gets called. Maybe add additional checks to be safe.

```
var wasEventRecieverCalled = false;
eventBridge.setEventReceiver(EventChannel.EVENT, object : EventReceiver {
      override fun receive(channel: EventChannel, event: Event) {
          assertEquals(event, testEvent)
          wasEventRecieverCalled = true;
      }
})

....

assertTrue(wasEventRecieverCalled)
```

Reply:
I guess the assertEquals check inside receiver can work? Let me add that to safeguard this.